wiki/People/Quality_Assurance_Team/en.md

```diff
@@ -9,18 +9,18 @@
 [Ym]: /wiki/shared/Insane-m.png "Yes"
 
 Quality Assurance Team
-========================
+======================
 
-The **Quality Assurance Team**, commonly referred to as QAT, are the aptly-named judicators of quality control who will check recently qualified beatmaps for quality and playability, retracting their qualified status as a disqualification notice where appropriate. They form the bulwark of standard control and enforce the basic expectation of quality for all beatmaps that enter the ranking process.
+The **Quality Assurance Team**, commonly referred to as QAT, are the judges of quality control who will check recently qualified beatmaps for quality and playability, retracting their qualified status as a disqualification notice where appropriate. They form the main component of standard control and enforce the basic expectation of quality for all beatmaps that enter the ranking process.
```

Comment:
imo, the wording in the previous version sounded a lot better (even tho you only changed two words)